src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -279,7 +291,11 @@ public final void visitToken(DetailAST blockCommentNode) {
             }
 
             if (result.getParseErrorMessage() == null) {
-                processTree(result.getTree());
+                final DetailNode javadocTree = result.getTree();
+                if (doesCheckAcceptJavadocWithUnclosedHtmlTag()
+                        || !JavadocUtils.hasUnclosedTag(javadocTree)) {
```

Comment:
@romani 
> If it is referring to unclosed tags why isn't it named that too?

Shouldn't `doesCheckAcceptJavadocWithUnclosedHtmlTag` be renamed to be something like `doesCheckAcceptJavadocWithNonTightHtml`?